website/siteConfig.js

```diff
@@ -28,19 +28,51 @@ module.exports = {
   api: [
     {
       title: 'BaseTable',
-      path: '/api/basetable',
+      path: `/api/${`basetable`.toLowerCase()}`,
```

Comment:
why this change? HTML is even agnostic to capitalization

Reply:
Hmm. But if I change `basetable` to `BaseTable`, it shows a blank page. May be because of the following:
`./website/src/template/api.js`, line 24. 

````javascript
      const link = links.find(link => link.to === `/api/${name.toLowerCase()}`)
````
Where `links` have the form:

````javascript
[
  {
    "key": "BaseTable",
    "title": "BaseTable",
    "to": "/api/BaseTable"
  },
  {
    "key": "Column",
    "title": "Column",
    "to": "/api/column"
  },
  // ...
]
````

Reply:
Anyway, If we don't need to expose any other than the previous four, we don't need it.